applications/webapp/src/App.tsx

```diff
@@ -16,7 +16,7 @@ import Home from "./Pages/Home/home";
 
 export default function App() {
   return (
-    <BrowserRouter>
+    <BrowserRouter basename="/app">
```

Comment:
Nice! It would be great to have a dedicated unit test that captures this scenario (the one that was before).